docs/sources/alerting/_index.md

```diff
@@ -6,49 +6,47 @@ weight = 110
 
 # Alerts overview
 
-Alerts allow you to identify problems in your system moments after they occur. By quickly identifying unintended changes in your system, you can minimize disruptions to your services.
+Alerts allow you to know about problems in your systems moments after they occur. Robust and actionable alerts help you identify and resolve issues quickly, minimizing disruption to your services.
 
-Alerts consists of two parts:
+Alerts have four main components:
 
-- Alert rules - When the alert is triggered. Alert rules are defined by one or more conditions that are regularly evaluated by Grafana.
-- Notification channel - How the alert is delivered. When the conditions of an alert rule are met, the Grafana notifies the channels configured for that alert.
-
-Currently only the graph panel visualization supports alerts.
+- Alert rule - One or more conditions, the frequency of evaluation, and the (optional) duration that a condition must be met before notifying.
```

Comment:
should we make a distinction between alert rules and alerts (insances) perhaps?

Reply:
The instances are the output of the alerting system, where as these are the configuration components that define the behavior of it. We will need to give plenty of love to the topic of alert instances (they will need their own page, I think) but I don't think here is the correct place for that.